colorfield/validators.py

```diff
@@ -17,3 +17,50 @@
     _("Enter a valid hexa color, eg. #00000000"),
     "invalid",
 )
+
+COLOR_RGB_RE = re.compile(
+    # prefix and opening parenthesis
+    r"^rgb\("
+    # first number
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # second number
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # third number
+    r"(\d{1,3})"
+    # closing parenthesis
+    r"\)$"
+)
```

Comment:
This regex was a bit hard to read, so I've split it out on multiple lines. The comments feel a bit trival, but If black reformats all strings on a single line if I remove them.

Reply:
I agreee, splitting it on multiple preserves it readability.